api/test/unit/services/user-csv-export.service.spec.ts

```diff
@@ -140,6 +140,11 @@ describe('Testing user csv export service', () => {
                     isJurisdictionalAdmin: true,
                   },
                 },
+                {
```

Comment:
I didn't think the new role was supposed to be able to see users?

Reply:
This adds limited jurisdictional admins to the user list. It adds all users except for admin